homework/index.js

```diff
@@ -29,19 +30,79 @@
     });
     return elem;
   }
+  function app(repos) {
+    const root = document.getElementById('root');
+    const container = createAndAppend('div', root, {
+      class: 'container',
+    });
+    const header = createAndAppend('header', container, {
+      class: 'header',
+    });
+    createAndAppend('p', header, {
+      text: 'HYF Repositories',
+    });
+    const select = createAndAppend('select', header, {
+      id: 'list',
+    });
+    const div = createAndAppend('div', root, { class: 'list-div' });
+    const list = createAndAppend('ul', div, { class: 'list1' });
+    const listContributor = createAndAppend('ul', div, { class: 'list2' });
+    const newList = createAndAppend('li', list, { text: ' ' });
+    const newList1 = createAndAppend('li', list, { text: ' ' });
+    const newList2 = createAndAppend('li', list, { text: ' ' });
+    const newList3 = createAndAppend('li', list, { text: ' ' });
 
+    for (let i = 0; i < repos.length; i++) {
+      createAndAppend('option', select, { text: repos[i].name, value: i });
+    }
+
+    function leftHand() {
+      newList.innerHTML = `Repository: <a target=_blank href= ${repos[select.value].html_url}>${
+        repos[select.value].name
+      }</a>`;
+      newList1.innerText = `Description: ${repos[select.value].description}`;
+      newList2.innerText = `Forks: ${repos[select.value].forks}`;
+      newList3.innerText = `Updated: ${repos[select.value].updated_at}`;
+      const links = repos[select.value].contributors_url;
+      fetchJSON(links, (err, data) => {
+        if (err) {
+          createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+        } else {
+          let linksContributor = JSON.stringify(data, null, 2);
+          linksContributor = JSON.parse(linksContributor);
+          listContributor.innerText = ' ';
+          for (let i = 0; i < linksContributor.length; i++) {
+            listContributor.innerHTML += `<li><a target =_blank href= ${
```

Comment:
Again, don't forget the quotes ""!

Reply:
👍

Reply:
Thanks for your feedback! I will pay my attention to it in the next time. :) 👍

Reply:
Thanks for the replies to @paulbremer's comments  - very professional! @paulbremer Do you want to see improvements on the code still or can we close this PR?

Reply:
All good! @Nadoshka did the improvements in the week2 homework 🎉This PR can be closed.